codex/blockexchange/network/networkpeer.nim

```diff
@@ -79,7 +79,7 @@ proc broadcast*(b: NetworkPeer, msg: Message) =
 
   asyncSpawn sendAwaiter()
 
-func new*(
+proc new*(
```

Comment:
Why change this? Usually we don't want side effects?

Reply:
Not sure, I might've been testing if affected some of the compile issues. I'll revert it back as it's not.